### PR TITLE
[WIP] Feature/111 add email clients

### PR DESCRIPTION
**Pull request by Daidalos117**
Follow of https://github.com/includable/react-native-email-link/issues/111

### CODE REVIEW

File: src/ios.js [hunk 1]
```diff
@@ -116,6 +118,9 @@ const uriParams = {
   protonmail: {
     path: "compose",
   },
+  seznamemail: {
+    path: "mail",
```
Comment (Daidalos117) [inline src/ios.js:122]: don't know how to get this :/

**Review by tschoffelen — APPROVED** [review]
Looks good!

**Comment by Daidalos117** [comment]
Yeah I think we can merge it for now. We actually discussed and we will contact authors of the client if they wanna tell us how to open compose view, but for our use case it's actually not needed :) Will added later if they reply.

**Comment by tschoffelen** [comment]
Thanks! Published to NPM as: `react-native-email-link@1.14.2`

**Pull request merged**